Dockerfile

```
@@ -0,0 +1,22 @@
+FROM php:8.2
+
+WORKDIR /usr/src/novo-site
+
+COPY . .
+
+RUN php -r "copy('https://getcomposer.org/installer', 'composer-setup.php');" && \
+    php -r "if (hash_file('sha384', 'composer-setup.php') === 'e21205b207c3ff031906575712edab6f13eb0b361f2085f1f1237b7126d785e826a450292b6cfd1d64d92e6563bbde02') { echo 'Installer verified'; } else { echo 'Installer corrupt'; unlink('composer-setup.php'); } echo PHP_EOL;" && \
+    php composer-setup.php && \
+    php -r "unlink('composer-setup.php');" && \
+    mv composer.phar /usr/local/bin/composer
```

```suggestion
COPY --from=composer:2.6.5 /usr/bin/composer /usr/bin/composer
```

Assim dá uma simplificada no comando de instalação copiando apenas o composer.phar do container oficial e fixando o número da versão do composer para garantir que em um build futuro não teremos problemas com uma versão futura do Composer.